lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -3201,6 +3208,32 @@ status_code_e s1ap_mme_handle_handover_notify(
         hashtable_rc_code2string(h_rc));
 
     s1ap_dump_enb(target_enb);
+
+    // Send context release command to source eNB
+    // Retrieve the association ID for the eNB that UE was connected to
+    // (i.e., Source eNB) and pull the Source eNB record from s1ap state using
+    // this association
+    if ((enb_array = hashtable_ts_get_elements(&state->enbs)) != NULL) {
+      for (idx = 0; idx < enb_array->num_elements; idx++) {
+        source_enb = (enb_description_t*) (uintptr_t) enb_array->elements[idx];
```

Comment:
You can get the source_enb before line 3184, by using src_ue_ref_p, as is done inside `s1ap_remove_ue`, with:

`source_enb = s1ap_state_get_enb(state, src_ue_ref_p->sctp_assoc_id);`